Doc/whatsnew/3.14.rst

```diff
@@ -1400,6 +1400,7 @@ Deprecated
   * :c:func:`!_PyUnicode_AsString`: use :c:func:`PyUnicode_AsUTF8`.
   * :c:func:`!_Py_HashPointer`: use :c:func:`Py_HashPointer`.
   * :c:func:`!_Py_fopen_obj`: use :c:func:`Py_fopen`.
+  * :c:func:`!_PyLong_New`: use :c:func:`PyLongWriter_Create`.
```

Comment:
Sort the entry: move it before _PyLong_Sign() in the list :-) Same remark for the NEWS entry.